homework_1/01_FizzBuzz/fizzbuzz/fizzBuzz.go

```diff
@@ -0,0 +1,35 @@
+package fizzbuzz
+
+import (
+	"strconv"
+
+	"github.com/pkg/errors"
+)
+
+func GetFizzBuzzSolution(start int, end int) ([]string, error) {
+	if start > end {
+		return nil, errors.New("Start is greater than end.")
+	}
+
+	if start < 1 {
+		return nil, errors.New("Start is lower than 1.")
+	}
```

Comment:
Why would zero and negative numbers be invalid for FizzBuzz?

Reply:
I changed it anyway. :smile: Is it ready for merge?

Reply:
Yep!